api-errors.go

```diff
@@ -500,9 +500,9 @@ var errorCodeResponse = map[APIErrorCode]APIError{
 		Description:    "Cannot specify more than one suffix rule in a filter.",
 		HTTPStatusCode: http.StatusBadRequest,
 	},
-	ErrFilterPrefixValueInvalid: {
+	ErrFilterValueInvalid: {
 		Code:           "InvalidArgument",
-		Description:    "prefix rule value cannot exceed 1024 characters",
+		Description:    "filter rule value is too long (> 1024) or contains unsupported characters",
```

Comment:
> Wait this error string is s3 compatible?

we should output the same error string? this is s3 msg: `Size of filter rule value cannot exceed 1024 bytes in UTF-8 representation`